demo/src/app/components/+pagination/demos/disable-direction-links/disable-direction-links.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-pagination-disable-direction-links',
```

Comment:
selector name `demo-pagination-disable-direction-links`
component name `DemoPaginationDirectionLinksComponent`
please be consistent in naming